src/cljs/pyregence/components/map_controls/measure_tool.cljs

```diff
@@ -52,13 +52,21 @@
            [:p "Note: There is a +/- 0.5% Great-Circle calculation error."]]
           [lon-lat-position "Point One Location" (if @point-one @point-one [0 0])]
           [lon-lat-position "Point Two Location" (if @point-two @point-two [0 0])]
-          [:div {:style {:display         "flex"
-                         :flex-direction  "column"
-                         :margin-top      ".2rem"}}
-           (when (> @distance-between-points 0)
-             [:div {:style {:height 30 :margin ".2rem"}}
-              [:label {:style ($/padding "1px" :1)}
-               (str (cl-format nil "~,4f" @distance-between-points) " meters")]])
+          (when (> @distance-between-points 0)
+            [:strong {:style {:display         "flex"
+                              :justify-content "center"
+                              :margin          ".4rem"
+                              :font-size       "1.2rem"}}
+             (let [format (fn [number label] (str (cl-format nil "~,1f" number) " " label))
+                   meters  @distance-between-points
+                   miles (* meters 0.00062137)
+                   kilometers (* meters 0.001)
+                   mi (format miles "miles")
+                   km (format kilometers "kilometers")]
```

Comment:
We typically try to keep `let` bindings at the top-level. Can you move these to within the outer `r/with-let`?

Also, another stylistic thing we do is align columns within bindings like so:
```suggestion
             (let [format     (fn [number label] (str (cl-format nil "~,1f" number) " " label))
                   meters     @distance-between-points
                   miles      (* meters 0.00062137)
                   kilometers (* meters 0.001)
                   mi         (format miles "miles")
                   km         (format kilometers "kilometers")]
```